jetstream/analysis.py

```diff
@@ -27,14 +30,13 @@ class Analysis:
 
     project: str
     dataset: str
-    config: AnalysisConfiguration
```

Comment:
Pickling `AnalysisConfiguration` results in a bunch of errors. For example, there are issues with pickling the `Statistic`s.